directives/tabs.py

```diff
@@ -0,0 +1,129 @@
+# This implementation is based on the sphinxcontrib-contentui repository:
+# https://github.com/ulrobix/sphinxcontrib-contentui/
+# -*- coding: utf-8 -*-
+"""
+Tabs
+====
+
+Directive for creating tabs.
+
+.. content-tabs::
+
+  .. tab-container:: tab1
+    :title: Tab title one
+
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus hendrerit 
+    auctor quam at maximus. Phasellus ornare suscipit tortor et aliquet. Aliquam 
+    erat volutpat. Aliquam at orci vel nibh tincidunt lacinia. Cras gravida, 
+    mauris eget vulputate ullamcorper, turpis est commodo velit, consequat 
+    pulvinar augue mi malesuada metus. Maecenas ac diam et augue placerat 
+    faucibus. Nullam ut iaculis nisi.
+
+  .. tab-container:: tab2
+    :title: Tab title two
+
+    .. code-block:: rst
+    
+      Insert Your Code here.
+"""
+import os
+from docutils.parsers.rst import Directive, directives
+from docutils import nodes
+from docutils.statemachine import StringList
+from sphinx.util.fileutil import copy_asset
+from pkg_resources import resource_filename
+from sphinx.util import logging
+
+import aplus_nodes
+
+CSS_FILE = 'css/aplus_tabs.css'
+JS_FILE = 'js/aplus_tabs.js'
+assets_path = '_static'
+
+logger = logging.getLogger(__name__)
+
+
+class ContentTabsDirective(Directive):
```

Comment:
The class names feel unintuitive to me at first. I would think that a "container" contains all the tabs while one "tab" is just the tab. That's just me. Is this all copied from the external library (sphinxcontrib-contentui)?

Reply:
Continuing on https://github.com/apluslms/a-plus-rst-tools/pull/83#discussion_r500170379. The answer is yes. The concept here is similar to the concept utilised by bootstrap. They basically have the navigation tabs, and then they associate each tab to certain content by using ids. https://getbootstrap.com/docs/3.3/javascript/#markup